src/telemetry/types.ts

```diff
@@ -95,3 +112,1229 @@ export interface IInterpreterPackages {
     getPackageVersion(interpreter: PythonEnvironment, packageName: string): Promise<string | undefined>;
     trackPackages(interpreterUri: InterpreterUri, ignoreCache?: boolean): void;
 }
+
+export interface IEventNamePropertyMapping {
```

Comment:
ah the telemetry generator has to look here instead of index.ts for the telemetry information. That's why telemetry.md is empty.

Reply:
See this file here:
https://github.com/microsoft/vscode-jupyter/blob/0a9f6009cc4aca40d233ffa8f03a5c943e93e810/src/platform/tools/telemetryGenerator.node.ts#L336

Reply:
🍺 thanks!